src/main/java/com/distillery/intentsender/ui/MainToolWindow.java

```diff
@@ -191,8 +192,11 @@ public void showFlagsSelection(
 
 	@Override
 	public void displaySelectedFlags(@NotNull List<? extends IntentFlags> selectedFlags) {
-		flagsTextField.setText(selectedFlags.toString());
-	}
+        String text = selectedFlags.size() > 0
+                ? selectedFlags.toString()
+                : IntentFlags.NONE.toString();
+        flagsTextField.setText(text);
```

Comment:
Minor: I'd move flags conversion to a separate method
```suggestion
        flagsTextField.setText(buildFlagsString(selectedFlags));
```